CONTRIBUTING.md

```diff
@@ -1,187 +1,148 @@
     - Needs to be ignored to be consistent with black
-- E302 Expected 2 blank lines, found 0
-- F841 local variable is assigned to but never used
-    - Probably have to ignore because that’s how stencils give outputs
+- `E302: Expected 2 blank lines, found 0`
+- `F841: local variable is assigned to but never used`
+    - Must ignore because gt4py stencils return outputs in place (no return statement)
     - Can avoid if all stencil assignments use square brackets on the left
-        - out[0, 0, 0] or out[idx]/out[curr]/out[something]
+        - `out[0, 0, 0]` or `out[idx]/out[curr]/out[something]`
 
-Flake8 rules not listed will be enforced unless we find a need not to enforce them. Since documentation does go out of date, please consult the entry in `setup.cfg` for the most up-to-date requirements and update this document if you notice a difference.
+Flake8 rules not listed will be enforced unless we find a need not to enforce them.
+Since documentation does go out of date, please consult the entry in `setup.cfg` for the most up-to-date requirements and update this document if you notice a difference.
 
 ## Style
 
-The first version of the dycore was written with minimal metadata and typing, motivated
-primarily by matching the regression data produced by the Fortran version of the code
-using the numpy backend. We are now actively refactoring for performance with multiple
-backends (cpu and gpu), moving code that still does computations in Python into GT4py
-stencils and merging stencils together with the introduction of enabling features in
-GT4py (such as regions). While we do that, clarifying the operation of the model and what the variables are
-will both help make the model easier to read and reduce errors as we move around long
-lists of argument variables.
-
-Specifically, we want to start adding as makes sense:
-- Type hints on python functions (see fv3core/utils/typing.py and below)
-- More descriptive types on stencil definitions (fv3core/utils/typing.py)
-- Docstrings on outward facing python functions: describe what methods are doing, describe
-the intent (in, out, inout) of the function arguments
+The first version of the dycore was written with minimal metadata and typing, motivated primarily by matching the regression data produced by the Fortran version of the code using the numpy backend.
+We are now actively refactoring, moving code that still does computations in Python into GT4py stencils and merging stencils together with the introduction of enabling features in GT4py (such as regions).
+While we do that, clarifying the operation of the model and what the variables are will both help make the model easier to read and reduce errors as we move around long lists of argument variables.
+
+Specifically, we want to start adding the following where appropriate:
+- Type hints on Python functions (see [`utils/typing.py`](https://github.com/VulcanClimateModeling/fv3core/blob/master/utils/typing.py) and below)
```

Comment:
this address is incorrect. https://github.com/VulcanClimateModeling/fv3core/blob/master/fv3core/utils/typing.py

Reply:
```suggestion
- Type hints on Python functions (see [`utils/typing.py`](https://github.com/VulcanClimateModeling/fv3core/blob/master/fv3core/utils/typing.py) and below)
```